ci-operator/config/openshift-priv/ovn-kubernetes/openshift-priv-ovn-kubernetes-release-4.20.yaml

```diff
@@ -414,6 +414,7 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "24"
       COMPUTE_NODE_TYPE: m5.xlarge
+      ENABLE_LAYER_3: "false"
       FEATURE: NetworkSegmentation
       ITERATION_MULTIPLIER_ENV: "3"
       KUBE_BURNER_VERSION: 1.5.0
```

Comment:
Remove this.

Reply:
Removed ENABLE_LAYER_3 & KUBE_BURNER_VERSION